lang/en/texts/nutriscore.html

```diff
@@ -45,7 +34,7 @@ <h2>How can I see the Nutri-Score of food products?</h2>
   <div class="small-12 medium-6 columns">
 
 
-<p>The Nutri-Score appears on the label of food products only in some countries and for a currently limited number of brands, but it is computed for all products of all countries on Open Food Facts since 2014!</p>
+<p>TNutri-Score is a voluntary labeling system. It appears on the label of food products only in some countries and for brands which commited to the score, but it is computed for all products of all countries on Open Food Facts since 2014!</p>
```

Comment:
```suggestion
<p>Nutri-Score is a voluntary labeling system. It appears on the label of food products only in some countries and for brands which commited to the score, but it is computed for all products of all countries on Open Food Facts since 2014!</p>
```